src/js/components/Table/Paginated/PaginatedTable.stories.tsx

```diff
@@ -396,7 +396,7 @@ function mapData(item) {
     birthdate: item.birthdate ? item.birthdate.toLocaleDateString('en-US') : 'undefined',
     age: `${item.age}`,
     programs: item.activePrograms.map((p, idx) => <div key={idx}>{p.name}</div>),
-    team: item.team.name,
+    team: item?.team?.name,
```

Comment:
```suggestion
    team: item.team?.name,
```

We dont need the first null check operator. I know it's just a `?` but theres a little bit of overhead added in the compiled JS everytime we use that, so using it unnecessarily is something we want to avoid :)